pandas/core/arrays/base.py

```diff
@@ -2337,6 +2337,19 @@ def _add_logical_ops(cls) -> None:
         setattr(cls, "__xor__", cls._create_logical_method(operator.xor))
         setattr(cls, "__rxor__", cls._create_logical_method(roperator.rxor))
 
+    def round(self, decimals: int = 0, *args, **kwargs) -> Self:
```

Comment:
Should I move this to the ExtensionArray class instead of ExtensionOpsMixin? This would allow SparseArray to use it.

Reply:
Yes.  If we expect it to exist, it should just be on the base class.